### PR TITLE
Minervaen/helloworldtest

### PR DESCRIPTION
**Pull request by MinervaEn**
- Adding hello world test.
-  fixing duplicated data-testid name under actioncreatoreditor modal.

### CODE REVIEW

File: cypress/integration/helloworld_test.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        .get('a[href$="/actions"]')
+        .click()
+  
+      // Click new action button
+      cy
+        .get('[data-testid="actions-button-create"]')
+        .click()
+  
+      // Click Action Type dropdown
+      cy
```
Comment (mattmazzola) [inline cypress/integration/helloworld_test.ts]: This seems unnecessary. It re-selects what was already selected.

File: cypress/integration/helloworld_test.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        .get('[data-testid="app-index-title"]')
+        .should(el => {
+          expect(el).to.contain(newAppName)
+        })
+    })
+  
+    /**
+     * FEATURE:  New Identity
+     */
+    it('it should create a NEW ENTITY', () => {
```
Comment (mattmazzola) [inline cypress/integration/helloworld_test.ts]: `it('it` is redundant. Remove the second one that's part of test description

File: cypress/integration/helloworld_test.ts [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  
+      // Click Action Type dropdown
+      cy
+        .get('[data-testid="dropdown-action-type"]')
+        .click()
+        .click()
+  
+      // Enter action response phrase: Hello World!
+      cy
+        .get('div[data-slate-editor="true"]')
```
Comment (mattmazzola) [inline cypress/integration/helloworld_test.ts]: This should be more specific to ensure you get the correct element in cases where the are multiple.  Also, would prefer not to couple selector to implementation detail of slate.  In other words, do not rely on that attribute.  Should only know about `contenteditable`

**Comment by MinervaEn** [comment]
@mattmazzola thanks for your PR review, I will address you comments and update the test file accordingly. Also I will adjust to and follow the team's contribution's practices.

**Pull request merged**